contracts/interfaces/ILightClient.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity ^0.8.0;
+
+interface ILightClient {
+    function initEpoch(
+        address[] memory _epochSigners,
+        uint256[] memory _epochVotingPowers,
+        uint256 height,
+        bytes32 headHash
+    ) external;
+
+    function submitHead(
+        uint256,
```

Comment:
Why the first parameter is not used?

Reply:
The first argument is provided here because the function will be overridden in the W3qProve contract and the first argument will be used.  You can see the instance with https://github.com/QuarkChain/staking-contracts/pull/5/files#diff-cde4e31d8242bcb343ab10ee753cb3aa34c031976604ef9a4fbfcc4645ef07f9

Reply:
I take a look at the Web3Prove code, where it looks like the first parameter `height` can be obtained from `BlockDecoder.verifyHeader()` so there is no need to have the parameter in the function?

<img width="511" alt="image" src="https://user-images.githubusercontent.com/2541286/166809593-ecc57bca-eec5-4c15-8724-0dfdc86f70e6.png">